cmd/minishift/cmd/start.go

```diff
@@ -76,6 +76,7 @@ var (
 	registryMirror   []string
 	openShiftEnv     []string
 	shellProxyEnv    string
+	proxyUri         string
```

Comment:
@budhrg sorry did not get your suggestion. You mean URI should be in capital letters?